This patch fixes the problem with forwarding in knot-resolver v4.3.0.
It reintroduces a fix which enables  policy related hack (knot/knot-resolver#205 (comment 94566) )
--- a/modules/policy/policy.lua
+++ b/modules/policy/policy.lua
@@ -982,7 +982,7 @@ policy.layer = {
 		if bit.band(state, bit.bor(kres.FAIL, kres.DONE)) ~= 0 then return state end
 		local qry = req:initial() -- same as :current() but more descriptive
 		return policy.evaluate(policy.rules, req, qry, state)
-			or (special_names_optim(req, qry.sname)
+			or (true--special_names_optim(req, qry.sname)
 					and policy.evaluate(policy.special_names, req, qry, state))
 			or state
 	end,
